kong/plugins/hmac-auth/access.lua

```diff
@@ -98,7 +99,8 @@ local function load_secret(username)
       local keys, err = dao.hmacauth_credentials:find_by_keys { username = username }
       local result
       if err then
-        return responses.send_HTTP_INTERNAL_SERVER_ERROR(err)
+        ngx_log(ngx.ERR, err)
+        return responses.send_HTTP_INTERNAL_SERVER_ERROR("Internal system error")
```

Comment:
Here it is better to give the `err` parameter. It will show the actual error in the logs for the user, but the consumer will only see a "Internal Server Error" body.